packtools/sps/validation/article_authors.py

```diff
@@ -0,0 +1,53 @@
+from packtools.sps.models.article_authors import Authors
+
+
+class ArticleAuthorsValidation:
+    def __init__(self, xmltree):
+        self._xmltree = xmltree
+        self.article_authors = Authors(self._xmltree)
+
+    def validate_authors_role(self, content_type_url):
```

Comment:
Colocar comentário para ficar explícito o que é content_type_url

Reply:
Entendi que `content_type_url` seria a lista de valores esperados de content-type e text.

Aí é uma questão de negócio que não deixei claro: os valores esperados eles são como se fossem parâmetros fixos (constantes) do sistema cliente, como se fossem uma "nota de corte" que, ao longo do tempo ela pode mudar, por isso não será uma constante no packtools e sim um parâmetro na função de validação.
Como "nota de corte" significa que todos os documentos ou grupos de documentos serão avaliados diante do mesmo parâmetro. 

Ou seja, nesta validação, o valor de `content_type_url` é a lista de URI e Text atual divulgada no site CRediT.
A validação que ter que ser feita é para cada role, verificar se URI e Text estão presentes em `content_type_url`.

Reply:
Entendi. No caso content_type_url, seria um dict com as chaves term e uri contendo todos os valores obtidos em: https://jats4r.org/credit-taxonomy#table-1-credit-terms-and-urls